content/pages/download-va-letters.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Download VA Letters
+layout: page-breadcrumbs.html
+---
+<script src="https://standards.usa.gov/assets/js/vendor/uswds.min.js" type="text/javascript"></script>
+
+<div class="main">
+<div class="section">
+<div class="row" markdown="1">
+
+# Download Your VA Letters
+
+<div class='va-introtext'>
+To receive some benefits, Veterans need a letter proving their status. You can access and download some of these benefit letters and documents online.
+</div>
+
+### What types of VA letters can I download using this tool?
+
+You can download your VA letters for benefits you’re currently receiving. You can also download letters for benefits you’re eligible for based on your service history.
+
+<a href='/download-va-letters/letters'><button class='usa-button-primary'>Access Your VA Letters</button></a>
```

Comment:
We can remove the button here and use it's class to style the anchor to look the same:
```
<a href='/download-va-letters/letters' class='usa-button-primary'>Access Your VA Letter</a>

```